### ISSUE TITLE
[Accessibility] MAS 4.2.8: (eclipse/split button '') An element of the given ControlType must support the ExpandCollapse pattern.

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4289]
The following accessibility issue needs investigation.

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS4.2.8; #ErrorSuggestion; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; #AI4D; #A11yAuto  

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**MAS Violated:** MAS 4.2.8

**App:** eclipse

**Element path:** split button ''

**Issue Details:** An element of the given ControlType must support the ExpandCollapse pattern. [Section 508 502.3.10](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** 1. Make sure the element has the appropriate ControlType property for its function.
2. If the current ControlType is correct, modify the element to support the ExpandCollapse pattern.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**MAS Reference:**
[MAS 4.2.8](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BBA368040-90A5-43C4-89A2-45F22E00E4E7%7D&file=MAS%204.2.8%20%E2%80%93%20List%20of%20Available%20Actions.docx&action=default&mobileredirect=true)

![Bug_4289_Java Tooling_Source tab_AI4D](https://user-images.githubusercontent.com/41108985/79009971-76f3b400-7b7e-11ea-85de-a5f280cb002b.png)

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Scala Plugin. You can find the same dialog in New Project > Scala Wizards > Scala Project. So it's out of the scope of our toolkit. Please find the screenshot of Eclipse\ScalaPlugin without our toolkit.
![image](https://user-images.githubusercontent.com/36756024/81463188-15a91a00-91ea-11ea-9c1e-a1d649e5d20d.png)

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Minimal. Corner case not blocking create project.
+ Quality of experience: Not Impacted. The combo-box could expand when clicking, not blocking create process.
+ Alternate solutions: This issue will not break users experience, no need for alternate solutions
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562987

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749645
